wiki/Tournaments/ATS/2018/en.md

```diff
@@ -28,55 +28,32 @@ The **Asian Taiko Showdown 2018 _(ATS 2018)_**, is a double elimination one-on-o
 ## Prizes
 
 | Placing | Prize(s) |
-| --- | :-- |
-| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | Special profile badge, Userpage banner |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 1 month of osu!supporter tag & userpage banner |
 | ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | Userpage banner |
 | ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | Userpage banner |
 
- *The remaining prizes are to be decided.*
-
-## Organization
+## Organisation
 
 The Asian Taiko Showdown 2018 is run by various osu!taiko community members.
```

Comment:
was run

Reply:
wait are we doing past tense now?

Reply:
for this sentence, yes; i think we was doing this in every official tournament

Reply:
See #1520. It's not applied at all.